wayland-backend/src/rs/server_impl/client.rs

```diff
@@ -199,7 +210,6 @@ impl<D> Client<D> {
 
         // Handle destruction if relevant
         if message_desc.is_destructor {
-            self.map.remove(object_id.id.id);
```

Comment:
why is this line removed?

Reply:
Unless I'm missing something, the line is redundant since `send_delete_id` also calls `self.map.remove`.

Reply:
Ah yes right, good point.